test/e2e/common/setup-kubestellar.sh

```diff
@@ -17,18 +17,23 @@ set -x # echo so users can understand what is happening
 set -e # exit on error
 SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 
+if [[ "$KFLEX_DISABLE_CHATTY" = true ]] ; then
+   disable_chatty_status="--chatty-status=false"
+   echo $disable_chatty_status
```

Comment:
I am surprised this works --- giving a command line flag to `echo` and `echo` does not interpret it.
If you wanted a more robust solution against changes in `echo` behavior, I would use a command more like

```
echo Disabling chatty status
```

or

```
echo "disable_chatty_status = $disable_chatty_status"
```

Reply:
I am no echo expert, but I tend to use echo to print environment variables, and on my setup they get printed as text or as empty string if not set. If this is not a standard, then I'll switch to your suggestion.  On a related topic, your LGTM has been received (and thank you for that) but github still waits approval label.  I wonder why?  In any case my changes for this comment will require a new LGTM.

Reply:
My concern here is not what happens when `$disable_chatty_status` expands to the empty string but rather what happens when that expands to `="--chatty-status=false`.  In that case, the bash command is `echo --chatty-status=false`. What does the `echo` command do with long flags (such as `--chatty-status`) that it does not recognize? A _normal_ command would fail with an error message.

Reply:
Getting a PR merged here requires BOTH an LGTM and an "approval". I deliberately gave only one so that you would have a choice in how to proceed.

Reply:
I changed the code to 
```
if [[ "$KFLEX_DISABLE_CHATTY" = true ]] ; then
   disable_chatty_status="--chatty-status=false"
   echo "disable_chatty_status = $disable_chatty_status"
fi
```

Reply:
Anything else that's needed before "approval" ?